composables/useChain.ts

```diff
@@ -24,15 +25,15 @@ export default function () {
   })
 
   const availableChains = computed(() => {
-    const availableUrlPrefixes = $store.getters['availableUrlPrefixes']
+    const chainList = availablePrefixes()
 
     if (!$config.public.dev) {
-      return availableUrlPrefixes.filter(
-        (urlPrefix) => !getChainTestList().includes(urlPrefix.value)
+      return chainList.filter(
+        (urlPrefix) => !getChainTestList().includes(`${urlPrefix.value}`)
```

Comment:
```suggestion
        (urlPrefix) => !getChainTestList().includes(urlPrefix.value)
```

Reply:
yep, it appeared a few minutes later. 
maybe wrap urlPrefix.value in a `String()` or type it `as string`